frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingRad.tsx

```diff
@@ -0,0 +1,41 @@
+import { Next } from "@navikt/ds-icons";
+import { BodyShort } from "@navikt/ds-react";
+import { Tiltaksgjennomforing } from "mulighetsrommet-api-client";
+import styles from "./Tiltaksgjennomforing.module.scss";
+import { useSideForNavigering } from "../../hooks/useSideForNavigering";
+
+interface TiltaksgjennomforingRadProps {
+  tiltaksgjennomforing: Tiltaksgjennomforing;
+  fagansvarlig?: boolean;
+}
+
+export function TiltaksgjennomforingRad({
+  tiltaksgjennomforing,
+  fagansvarlig = false,
+}: TiltaksgjennomforingRadProps) {
+  const side = useSideForNavigering();
+  return (
+    <li
+      className={styles.rad}
+      onClick={() =>
+        (location.href = fagansvarlig
+          ? `${side}/${tiltaksgjennomforing.tiltakstype.id}/tiltaksgjennomforing/${tiltaksgjennomforing.id}`
+          : `${side}/tiltaksgjennomforing/${tiltaksgjennomforing.id}`)
+      }
```

Comment:
For å gjøre denne litt mer selvforklarende kunne du dratt onClick-funksjonaliteten ut i egen funksjon og kalt feks `navigerSomFagEllerTiltaksansvarlig`. Da blir koden litt mer selvforklarende og lett å forstå senere

Reply:
Og burde ikke li-elementet egentlig hatt en lenke inni seg, istedenfor onClick? For UU og korrekt semantisk html?